/*
 * Copyright (C) 1996-2020 The Squid Software Foundation and contributors
 *
 * Squid software is distributed under GPLv2+ license and includes
 * contributions from numerous individuals and organizations.
 * Please see the COPYING and CONTRIBUTORS files for details.
 */

--- /usr/include/kerberosv5/krb5.h.orig	Mon Aug 23 04:09:23 2010
+++ /usr/include/kerberosv5/krb5.h	Mon Aug 23 04:10:53 2010
@@ -171,6 +171,7 @@
 #define KRB5INT_END_DECLS
 #endif
 #endif
+KRB5INT_BEGIN_DECLS
 
 #if TARGET_OS_MAC 
 #    pragma options align=mac68k 
@@ -3246,8 +3247,4 @@
 /* for compatibility with older versions... */
 #define asn1_err_base ERROR_TABLE_BASE_asn1
 
-#ifdef __cplusplus
-}
-#endif
-
 #endif		/* _KRB5_H */
